app/controllers/admin/document_collection_groups_controller.rb

```diff
@@ -47,6 +47,17 @@ def update
     render_design_system(:edit, :legacy_edit)
   end
 
+  def reorder; end
```

Comment:
For both reorder, and order, do we need to also add some forbidden checks unless users have ability to preview design system? i feel like with this, people can just access the order page through URL regardless of permissions

Reply:
Yes, that makes sense. I've added that for both of those (think it's working correctly)